src/auditwheel/manylinux.rs

```diff
@@ -0,0 +1,47 @@
+use serde::{Deserialize, Serialize};
+use std::fmt;
+use std::str::FromStr;
+
+/// Decides how to handle manylinux compliance
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
```

Comment:
I was think about making this enum `Copy` since it's cheap to do.

Reply:
Good idea